iscript/src/iscript/mac.py

```diff
@@ -165,9 +165,7 @@ async def sign_single_files(config, sign_config, all_paths):
 
     for app in all_paths:
         app.check_required_attrs(["orig_path", "parent_dir", "artifact_prefix", "single_file_globs"])
-        app.target_tar_path = "{}/{}{}".format(config["artifact_dir"], app.artifact_prefix, app.orig_path.split(app.artifact_prefix)[1]).replace(
-            ".zip", ".tar.gz"
-        )
+        app.target_tar_path = "{}/{}{}".format(config["artifact_dir"], app.artifact_prefix, app.orig_path.split(app.artifact_prefix)[1])
```

Comment:
Can we rename this variable, given that it's not always a tarball now?

Reply:
I thought about that. `target_bundle_path` ?

Reply:
wfm